Kickstarter-iOS/Features/ProjectPage/Controller/ProjectPageViewController.swift

```diff
@@ -989,7 +989,11 @@ extension ProjectPageViewController: ProjectPamphletMainCellDelegate {
     _ cell: ProjectPamphletMainCell,
     goToCreatorForProject project: Project
   ) {
-    guard AppEnvironment.current.currentUser != nil, featureBlockUsersEnabled() else {
+    guard
+      AppEnvironment.current.currentUser != nil,
+      featureBlockUsersEnabled(),
+      !project.creator.isBlocked
```

Comment:
I wonder if this is necessary since users shouldn't be able to view projects by blocked creators.

Reply:
The projects won't show up in search or discover, but they could still get there from a deeplink. Or if they have a message history with the creator (unusual, but possible), then they can navigate there from messages.